src/test/java/autoracing/LocationTest.java

```diff
@@ -0,0 +1,31 @@
+package autoracing;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LocationTest {
+    @Test
+    public void testStartingLine() {
+        Location startingLine = Location.STARTING_LINE;
+        assertThat(startingLine.getDistance()).isEqualTo(0);
+        assertThat(startingLine.getRound()).isEqualTo(0);
+    }
+
+    @Test
+    public void testValueObject() {
```

Comment:
valueObject를 테스트한다기보단, StartingLine.move()를 테스트하는것 같은데
```suggestion
    public void testMove() {
```
는 어떨까요??